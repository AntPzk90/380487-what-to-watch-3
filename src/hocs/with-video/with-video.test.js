import React from 'react';
import renderer from 'react-test-renderer';
import withVideo from './with-video.jsx';

const isPlayMock = false;
const isPauseMock = false;
const isAutoplayMock = true;
const isMutedMock = true;

it(`SnapshotTest withVideo`, () => {

  const tree = renderer
  .create(
      <withVideo
        isPlay = {isPlayMock}
        isPause = {isPauseMock}
        isAutopla = {isAutoplayMock}
        isMutedMock = {isMutedMock}
      />
  )
  .toJSON();

  expect(tree).toMatchSnapshot();
});
