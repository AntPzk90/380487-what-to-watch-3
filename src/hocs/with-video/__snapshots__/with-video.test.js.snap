// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapshotTest withVideo 1`] = `
<withVideo
  isAutopla={true}
  isMutedMock={true}
  isPause={false}
  isPlay={false}
/>
`;
